packages/react-components/src/components/utils/responsive.ts

```diff
@@ -78,7 +78,7 @@ export const _useContainerHeight = (containerRef: RefObject<HTMLElement>): numbe
   return height;
 };
 
-const NARROW_WIDTH_REM = 30;
+const NARROW_WIDTH_REM = 27.5;
```

Comment:
We should be updating the logic in the mediaGallery to make sure when on dekstop we are forcing the videoGallery to not trip is narrow and to always be the desktop mode. there should be API's and logic for this.